### ISSUE TITLE
Change the default/empty arc color from rgb(92,92,92)

### ISSUE THREAD

**Issue by antoniolago** [antoniolago/react-gauge-component#39]
### Discussed in https://github.com/antoniolago/react-gauge-component/discussions/26

<div type='discussions-op-text'>

<sup>Originally posted by **buildpath-ian** November  6, 2023</sup>
How can I modify the color of the dark grey default arc ?</div>

**Comment by notwatermango** [comment]
I've been looking for this too, need at least a light mode grafana color.

**Comment by LucasPupat** [comment]
In a CSS file: 
```
.doughnut > :last-child > path {
    fill: #f00 !important;
}
```

**Comment by antoniolago** [comment]
Implemented on v1.2.3
